### ISSUE TITLE
Troubles running with ASAN

### ISSUE THREAD

**Issue by domenukk** [AFLplusplus/AFLplusplus#290]
When compiling afl++ 33dd55 with `ASAN_BUILD=1 make`, the forkserver start fails.
Target was compiled with afl-gcc. 
It works fine when compiled without.
Debug output with `AFL_DEBUG_CHILD_OUTPUT = 1`:
```bash
[*] Spinning up the fork server...
[+] Loaded environment variable AFL_DEBUG_CHILD_OUTPUT with value 1
=================================================================
ERROR: Failed to mmap
[+] All right - fork server is up.

[-]  SYSTEM ERROR : Unable to request new process from fork server (OOM?)
    Stop location : run_target(), src/afl-fuzz-run.c:57
       OS message : Broken pipe
```

**Comment by vanhauser-thc** [comment]
can you check if this is fixed with a5c09e4 ?

**Comment by domenukk** [comment]
Nope and the ASAN output is not printed. I'll see why.
`-m none` works, so once the check works, it should be fine.

**Comment by domenukk** [comment]
Fixed in ba4f018
